libsolidity/interface/StandardCompiler.cpp

```diff
@@ -728,17 +729,17 @@ std::variant<StandardCompiler::InputsAndSettings, Json::Value> StandardCompiler:
 	}
 	else if (ret.language == "SolidityAST")
 	{
-		for (auto const& sourceName: sources.getMemberNames())
+		for (auto const& [sourceName, _]: sources.items())
```

Comment:
I'm by the way a bit surprised that the compilers don't complain about ``_`` being unused in these cases, but if it's fine then good :-).

Reply:
Yeah its quite cool :)